app/controllers/api/v2/hosts_controller.rb

```diff
@@ -126,6 +126,13 @@ def destroy
         process_response @host.destroy
       end
 
+      api :GET, "/hosts/:id/enc", N_("Get ENC values of host")
+      param :id, :identifier_dottable, :required => true
+
+      def enc
+        render :json => { :data => @host.info }.to_json if @host
```

Comment:
Isn't the `.to_json` here redundant? `render :json => ` will already transform the hash to JSON I think